src/avram/migrator/migration.cr

```diff
@@ -96,16 +96,6 @@ abstract class Avram::Migrator::Migration::V1
         yield tx
       end
     end
-  rescue e : PQ::PQError
```

Comment:
Any idea if the actual stack trace will include the statement that failed? I'm wondering if instead of removing this one, we just figure out how to add the filename to it. I'm worried letting it error on its own wouldn't really provide that nice of an error.

Reply:
So it wouldn't. One thing I was thinking about is handling all errors that are raised from migrations, wrapping them with more information (name of migration, statement that caused it), and printing them ourselves so that we can include the cause of exceptions and then exiting as a failure. What do you think?

Reply:
Calling https://crystal-lang.org/api/0.35.1/Exception.html#inspect_with_backtrace:String-instance-method on exceptions includes the cause, I don't know why Crystal doesn't use it in the unhandled exception code flow.

Reply:
Yeah, I like that. One thing that sets us apart from others is how we display really nice error messages. Do you want to do that in this PR, or a different one? I'm cool with either way.

Reply:
I'll update this PR. I believe this will help https://github.com/luckyframework/avram/pull/578#issuecomment-751812307 as well

Reply:
@jwoertink done 👍  this is much better, I think